netlify.toml

```diff
@@ -21,3 +22,8 @@ force = true
 from = "/*"
 to = "/404.html"
 status = 404
+
+[[redirects]]
```

Comment:
Can you give some info on what this redirect does?

Reply:
Removed this!